### PR TITLE
Edit, destroy task

### PR DESCRIPTION
**Pull request by NgocVu1994**

### CODE REVIEW

File: app/views/user_tasks/show.html.erb [hunk 1]
```diff
@@ -12,23 +12,56 @@
...
+              <button id="dLabel" type="button" data-toggle="dropdown"
+                aria-haspopup="true" aria-expanded="false"
+                class="btn btn-default">
+                <span class="glyphicon glyphicon-align-justify"
+                  aria-hidden="true"></span>
+              </button>
+              <ul class="dropdown-menu dropdown-menu-right"
+                aria-labelledby="dLabel">
+                <li>
+                  <a href="<%= edit_course_subject_task_path(@course_subject, @user_task.task) %>">
```
Comment (truonglocbinh) [inline app/views/user_tasks/show.html.erb]: remove ()

File: app/views/user_tasks/show.html.erb [hunk 2]
```diff
@@ -12,23 +12,56 @@
...
+                    <i class="fa fa-pencil" aria-hidden="true"></i>
+                    <%= t"courses.buttons.admin.edit" %>
+                  </a>
+                </li>
+                <li>
+                  <a href="<%= course_subject_task_path(@course_subject, @user_task.task) %>",
```
Comment (truonglocbinh) [inline app/views/user_tasks/show.html.erb]: remove()

**Comment by truonglocbinh** [comment]
ok

**Comment by vietmt** [comment]
Ok

**Pull request merged**